api/src/utilities/unit-groups-transformations.ts

```diff
@@ -55,21 +60,37 @@ export const getUnitGroupSummary = (
     let rentAsPercentIncomeRange: MinMax,
       rentRange: MinMax,
       amiPercentageRange: MinMax;
-    group.unitGroupAmiLevels.forEach((level) => {
-      if (
-        level.monthlyRentDeterminationType ===
-        MonthlyRentDeterminationTypeEnum.flatRent
-      ) {
-        rentRange = setMinMax(rentRange, level.flatRentValue);
+
+    if (!isNonRegulated) {
```

Comment:
suggestion: This would be a good function to add tests to in unit-groups-transformations.spec.ts. I think just one test needs to be added where `isNonRegulated` is set to true